README.md

```diff
@@ -156,14 +156,14 @@ First the good news: you will _not_ have to migrate your keychain data when upgr
 
 Now the bad news: the Swift Valet API has slight differences from the Objective-C Valet API. You may have noticed a few of the differences in the sample code above, but here's a rundown of the changes that may affect you.
 
-1. Initializers have changed in both Swift and Objective-C. [See examples above](#Basic-Initialization).
-2. `VALSynchronizableValet` has been replaced by a `Valet`  (or `VALValet` in Objective-C) with an `.iCloud` flavor. [See examples above](#Sharing-Secrets-Across-Devices-with-iCloud).
+1. Initializers have changed in both Swift and Objective-C - both languages use class methods now, which felt more semantically honest (a lot of the time you're not instantiating a new Valet, you're re-accessing one you've already created). [See example usage above](#Basic-Initialization).
+2. `VALSynchronizableValet` (which allowed keychains to be synced to iCloud) has been replaced by a `Valet.iCloudValet()`  (or `[VALValet iCloudValet]` in Objective-C). [See examples above](#Sharing-Secrets-Across-Devices-with-iCloud).
```

Comment:
`Valet.iCloudValet(with:accessibility:)` and `+iCloudValetWithIdentifier:accessibility:`?

Maybe? Not sure how to properly signify static w/ Swift like we do in Objc

Reply:
Sure, felt icky trying to shorthand tbh